pkg/master/controller.go

```diff
@@ -257,18 +260,20 @@ type EndpointReconciler interface {
 // masterCountEndpointReconciler reconciles endpoints based on a specified expected number of
 // masters. masterCountEndpointReconciler implements EndpointReconciler.
 type masterCountEndpointReconciler struct {
-	masterCount      int
-	endpointRegistry endpoint.Registry
+	masterCount       int
+	endpointRegistry  endpoint.Registry
+	configmapRegistry configmap.Registry
```

Comment:
There should be no changes in this file. Fix genericapiserver to read the configmap.